front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -173,6 +184,22 @@ const StdcmConfig = ({
     }
   }, []);
 
+  useEffect(() => {
+    if (isPathFindingLoading) {
+      setShowMessage(true);
+    }
+
```

Comment:
We need to add this case, otherwise the hidden banner is never removed from the DOM if the pathfinding fails.
```suggestion
    if (pathfinding?.status === 'failure') {
      setShowMessage(false);
    }
```
Also add `pathfinding?.status` in the deps array